.vscode/tasks.json

```diff
@@ -17,7 +17,9 @@
                 "showReuseMessage": false
             },
             "command": "./checks/run_meta_check.py",
+
             "problemMatcher": {
+                "severity": "info",
```

Comment:
As there is a severity field in the matcher, I don't see why the default severity is needed. Please can you elaborate on this :)?

Reply:
GitHub 'annotations' (the actions problem matcher) can be one of three severities, '`ERROR` `WARNING` or ***`NOTICE`***. All of the checks outputs are made first to conform to this schema. VSCode problem matchers also recognise three levels, `ERROR` `WARNING` and ***`INFO`***. This means any `NOTICE` level matches have their severity level ignored, and will use the default severity `ERROR`. Since everything else should have a severity level, only notices map to info and everything is as it should be.

TDLR; When this line is removed, all `INFO` level messages become `ERROR`'s

Reply:
Ah ok, it explains some things I have seen ERRORS when it should have been notices. Thank you for figuring this out :-).